src/fheroes2/agg/agg_image.cpp

```diff
@@ -1306,16 +1295,27 @@ namespace
             else if ( id == ICN::BUTTON_LOAD_MAP_EVIL ) {
                 goodButtonIcnID = ICN::BUTTON_LOAD_MAP_GOOD;
             }
+            else if ( id == ICN::BUTTONS_NEW_GAME_MENU_EVIL ) {
+                goodButtonIcnID = ICN::BUTTONS_NEW_GAME_MENU_GOOD;
+            }
+            else if ( id == ICN::BUTTONS_EDITOR_MENU_EVIL ) {
+                goodButtonIcnID = ICN::BUTTONS_EDITOR_MENU_GOOD;
+            }
+            _icnVsSprite[id].resize( fheroes2::AGG::GetICNCount( goodButtonIcnID ) );
+            const size_t icnSize = _icnVsSprite[id].size();
+            for ( size_t i = 0; i < ( icnSize / 2 ); ++i ) {
+                convertToEvilButtonBackground( _icnVsSprite[id][i * 2], _icnVsSprite[id][i * 2 + 1], goodButtonIcnID, static_cast<uint32_t>( i * 2 ) );
+            }
 
-            convertToEvilButtonBackground( _icnVsSprite[id][0], _icnVsSprite[id][1], goodButtonIcnID );
             break;
         }
         case ICN::BUTTONS_FILE_DIALOG_EVIL:
         case ICN::BUTTONS_FILE_DIALOG_GOOD: {
             _icnVsSprite[id].resize( 8 );
 
+            const bool isEvilInterface = id == ICN::BUTTONS_FILE_DIALOG_EVIL;
```

Comment:
For better readability we can add parentheses:
```suggestion
            const bool isEvilInterface = (id == ICN::BUTTONS_FILE_DIALOG_EVIL);
```